dev-meetings/2025/2025-07-04/README.md

```diff
@@ -1,8 +1,96 @@
 # Gammapy Developer Meeting 
- * Friday, July 04, 2025, at 2 pm (CET) 
- * Gammapy Developer Meeting on Zoom (direct link on Slack) 
+ * Friday, July 04, 2025, at 2 pm (CET) - 15:14 pm
+ * Gammapy Developer Meeting on Zoom (direct link on Slack)
+ * Attendees: Régis Terrier (R.T.), Atreyee Sinha (A.S.), Tomas Bylund (B.S.), Samantha Wong (S.W.), Katharina Egg (K.E.), Mireia Nievas Rosillo (M.N.R.), Leander Schlegel (L.S.)
+
 # Agenda
 
+* Time for codefreeze (branch that contains v2, stop introducing features):   
+AS: Before September, R.T.: Decide on time next week. Initially next Friday, prob. postponed.
+
+* Contrib. Samantha Wong (S.W.):
+RT: Tutorial content fine, move new function to utils.regions. 
+SW: Could make function flexible, but needed or better specific? 
+RT: Not decided if instrument specific functions into GP, so far not. Rely on regions for the function instead of utils. Extract_bright_star_region_list? 
+SW: Try to implement it today!
+A.S. question to R.T. regarding this: 
+S.W.: Regular exclusion code (notebook), extra function. For crab scenario very bright star falls in region.
+Try up to finish everything today in order for codefreeze.
+
+## Information
+
+## Recurring items
+
+R.T.: Did we assign someone for recurring items yet?
+
+### Open issues
+
+1. Issue: test class not properly initialized, corrected by A.S., merged today by R.T.
+2. Issue: Deprecation, R.T.: PR already open for deprecation, still work todo in this PR -> but issue can be removed.
+3. Issue: Repetitive wording of recurring exercises, A.S. did PR to remove this section. -> can be closed.
+4. Issue: TSMapEstimator failing, Q.R. open PR to solve it, wait for Lauras feedback. A.S. tested it, Q.R. iterated, seems to work -> R.T. Wait for Lauras confirmation.
+5: Issue: already discussed, not relevant for now
+
+### Bugs
+
+A.S. made list showing the status. Question regarding which bugs to fix dependent on label, R.T. Is postponing a problem, but sim for data challenge in first part of 2026, thereby solve later ok. A.S. asks Fabio (?)
```

Comment:
```suggestion
A.S. made list showing the status. Question regarding which bugs to fix dependent on label, R.T. proposes to postpone [#5549](https://github.com/gammapy/gammapy/issues/5549) since simulations for data challenge should take place in first part of 2026, thereby solve later ok. A.S. asks Fabio to confirm.
```